client/src/browser/components/BrowserWrapper.tsx

```diff
@@ -17,18 +22,45 @@ export const mapDispatchToProps = {
 
 type MergedProps = ReturnType<typeof mapStateToProps> & DispatchProps<typeof mapDispatchToProps>;
 
-const BrowserWrapper: React.SFC<MergedProps> = ({ isOpen, open }) => {
+const BrowserWrapper: React.SFC<MergedProps> = ({ isOpen, open, noOfDatasets, isVisible}) => {
     if (!isOpen) {
-        return (
-            <Button icon={true} labelPosition="left" onClick={open}>
-                <Icon name='add' />
-                Browse
-            </Button>
+        return ( 
+            <div>
+                <div style={{textAlign: 'center'}}>
+                    { (!noOfDatasets && !isVisible) ?
+                    <div>
+                        <h1>Get Started With LiberTEM</h1>
+                        <img src={libertem} alt="libertem" style={{paddingTop: '5%', paddingBottom: '5%', opacity: '0.6'}}/>
+                        <div style={{marginLeft: '30%', marginRight: '30%', paddingBottom: '5%'}}>
+                            <text>
+                            LiberTEM is an open source platform for high-throughput distributed processing of pixelated scanning transmission electron microscopy (STEM) data. Click on "Browse" to select the dataset you want to analyze.
+                            </text>
+                        </div>
+                    </div>
+                    : ''
+                    }
+                    <Button icon={true} labelPosition="left" onClick={open} color='blue'>
+                        <Icon name='add' />
+                        Browse
+                    </Button>
+                    { (!noOfDatasets && !isVisible) ? 
+                    <div style={{paddingTop: '5%', paddingBottom: '5%'}}>
+                        <a href={'https://libertem.github.io/LiberTEM/'} target='_blank'>Learn more about LiberTEM</a>
+                        <br/>
+                        <br/>
+                        <img src={github} alt="github" style={{height: '25px', width: '25px'}}/>
+                        <a href={'https://github.com/LiberTEM/LiberTEM'} target='_blank'> Star us on GitHub</a>
```

Comment:
External links with `target='_blank'` should always have `rel='noopener'`: https://mathiasbynens.github.io/rel-noopener/